Bottom: 108773485dfe27114cc1c92e2fbb97f7d16a0520
Top:    7788080f6e64d6d9c1d0d69376ba0f8917d40fb1
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 13:23:19 -0700

Refresh of remove-file-use-from

---
diff --git a/lldb/scripts/interface/SBCommandReturnObject.i b/lldb/scripts/interface/SBCommandReturnObject.i
index 38d9cea2214..73d4001aaba 100644
--- a/lldb/scripts/interface/SBCommandReturnObject.i
+++ b/lldb/scripts/interface/SBCommandReturnObject.i
@@ -99,7 +99,7 @@ public:
     %extend {
         // transfer_ownership does nothing, and is here for compatibility with
         // old scripts.  Ownership is tracked by reference count in the ordinary way.
- 
+
         void SetImmediateOutputFile(lldb::FileSP BORROWED, bool transfer_ownership) {
             self->SetImmediateOutputFile(BORROWED);
         }
